src/CruxComponent.tsx

```diff
@@ -81,27 +93,53 @@ export class CruxComponentCreator {
             }
 
             checkAdditionalModel(modelName: string) {
-                if((modelName === constants.modelName && constants.paginate) || 
-                    !Array.isArray(this.props.additionalModels[modelName])){
+                if ((modelName === constants.modelName && constants.paginate) ||
+                    !Array.isArray(this.props.additionalModels[modelName])) {
                     return true
                 }
                 return _.isEmpty(this.props.additionalModels[modelName])
             }
 
             fetchServerData(modelName: string) {
-                if(modelName === constants.modelName){
-                    this.getDefaultPageSize() ? this.props.filter(modelName, {limit : constants.paginate.defaultPageSize}) : this.props.fetch(modelName)
+                if (modelName === constants.modelName) {
+                    this.getDefaultPageSize() ? this.props.filter(modelName, { limit: constants.paginate.defaultPageSize }, this.searchById) : this.props.fetch(modelName, this.searchById)
                 } else {
                     this.props.fetch(modelName)
                 }
             }
 
+            searchById(data: any) {
+                const params = new URLSearchParams(this.props.location.search)
+                const searchId = params.get("id")
+                const searchField = params.get("field")
```

Comment:
const {searchId, searchFieldId}  = new URLSearchParams(this.props.location.search)